From fb4f68445a377ea8c702f02aea2a6bfc1c917752 Mon Sep 17 00:00:00 2001
From: Michael Marineau <mike@marineau.org>
Date: Sat, 20 Dec 2014 15:27:00 -0800
Subject: [PATCH] remove broken ez_setup import

---
 setup.py | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/setup.py b/setup.py
index fe3dbd1..d52670b 100644
--- a/setup.py
+++ b/setup.py
@@ -15,9 +15,6 @@
 
 import sys
 
-import ez_setup
-ez_setup.use_setuptools()
-
 from setuptools import setup, find_packages
 from setuptools.command import test
 
-- 
2.0.4

